tests/tekton-tasks_test.go

```diff
@@ -242,4 +245,41 @@ var _ = Describe("Tekton-tasks", func() {
 			}, tenSecondTimeout, time.Second).Should(BeTrue(), "there should be no role bindings left")
 		})
 	})
+	Context("multiple CRs deployed", func() {
+		AfterEach(func() {
+			// To reset TTO we need to delete it and deploy it again
+			tto := strategy.GetTTO()
+			apiClient.Delete(ctx, tto)
+			strategy.CreateTTOIfNeeded()
+			waitUntilDeployed()
+		})
+
+		It("check if correct status is set for TTO", func() {
+			tto1 := strategy.createTekton("tto-test-1")
```

Comment:
Move this into `BeforeEach`? And get rid of the additional `createTekton` method?